Include/internal/pycore_runtime.h

```diff
@@ -125,6 +125,12 @@ typedef struct _Py_DebugOffsets {
     } tuple_object;
 } _Py_DebugOffsets;
 
+/* Reference tracer state */
+struct _reftracer_runtime_state {
+    PyRefTracer tracer_func;
+    void* data;
```

Comment:
I suggest to either remove "tracer_" prefix from "tracer_func", or to add it to both.

```suggestion
    void* tracer_data;
```